skll/learner/__init__.py

```diff
@@ -125,79 +138,79 @@ class Learner(object):
         size of the learner object in memory and also when
         it is saved to disk.
         Defaults to ``False``.
-    feature_scaling : str, optional
+    feature_scaling : str
         How to scale the features, if at all. Options are
         -  'with_std': scale features using the standard deviation
         -  'with_mean': center features using the mean
         -  'both': do both scaling as well as centering
         -  'none': do neither scaling nor centering
         Defaults to 'none'.
-    model_kwargs : dict, optional
+    model_kwargs : Optional[Dict[str, Any]]
         A dictionary of keyword arguments to pass to the
         initializer for the specified model.
         Defaults to ``None``.
-    pos_label : str, optional
-        A string denoting the label of the class to be
+    pos_label : Optional[LabelType]
+        An integer or string denoting the label of the class to be
```

Comment:
can it be a float too? (like LabelType)

Reply:
Kind of since it can be an int and an int is a float technically :-p.